### ISSUE TITLE
Guild Houses en Mordor

### ISSUE THREAD

**Issue by armadaazteca** [Armada-Azteca/ArmadaAztecaServer#86]
Estos son los detalles de las guildhouses:

- [ ] Cada guildhouse tendra 4 respawns,
- [ ] Los respawn estaran agrupados de 2 en 2, cada grupo separado del otro.
- [ ] Los respawns, tendran goblins que se mueven
- [ ] Cada grupo de 2 respawn tendra una puerta, pero para llegar a ellos no sera casa, ni protection zone.

**Comment by Eduardox** [comment]
Una sola guild podrá ser dueña de las 2 guildhouse?

**Comment by armadaazteca** [comment]
No, porque una guild no se podra declarar guerra a si misma, ese sistema aun no lo pongo, Pero ya pondre los detalles en estos dias en un issue aparte.

**Comment by Eduardox** [comment]
Por ejemplo, ahorita solo hay 2 guilds (principales) las otras que hay (creo que solo hay otras 2) son de low levels (noobsillas), ponle que 1 guildhouse se le da a 1 de las 2 principales y la otra a la de los low level, es prob. que las 2 houses terminen en manos de las 2 guild principales del server, y ahi que pedo si se declaran war? Me refiero a que una sola guild terminaría quedandose las 2 casas no?

**Comment by RessivaFretza** [comment]
Como yo lo veria es que solo pueden declarar guerra las de mas abajo, para arriba! Asi conseguir la guild house y si la que tiene casa principal cuando declare y en un caso particular gane! que solo se le otorgue prestigio o estrellas o mas tiempo para mantener la casa! y asi podemos insitar a que creen mejores guilds! Otra propuesta podria ser que las guilds house fueran divididas 1 para las mejores guilds y otra para las guild noobs, el pedo es que si la guild principal manda a todos los noobs a una cantera alli si tendra las 2 casas!

**Comment by armadaazteca** [comment]
oviamente, nada que ver una casa con la otra, no seran iguales, una es mas chica y con menos respawn, tal vez esa con goblins quietos, entonces defiitivamente van a querer todos pelear por la primera y si la guld ganadora quiere hacer otra miniguild, pues corren el riesgo de que otra guild se arme y les quite una de las casas por dividirse.

El punto es que aunque haya dos guild principales, todos quieran pelear por la primer GH, y asi las mas pequeñas se podran poner a pelear por la otra. Espero yo que se armen mas guilds porque no todos los miembros de la guild podran levelar en la casa, tons les convendra, hacer otra guild e intentar ganarla.
